src/plugins/dashboard/public/dashboard_app/dashboard_router.tsx

```diff
@@ -100,7 +100,7 @@ export async function mountApp({ core, element, appUnMounted, mountContext }: Da
 
   const renderDashboard = (routeProps: RouteComponentProps<{ id?: string }>) => {
     const routeParams = parse(routeProps.history.location.search);
-    if (routeParams.embed && !globalEmbedSettings) {
+    if (routeParams.embed === 'true' && !globalEmbedSettings) {
```

Comment:
`routeParams.embed` is a string, so the previous logic would evaluate to `true` even if `embed=false` was in the URL because `Boolean("false")` resolves to `true`